src/sections/DiscoveryParagraph/DiscoveryParagraph.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FunctionComponent } from 'react';
+import { Container, Row, Col } from 'react-grid-system';
+
+import Elephant from '~/assets/img/elephant.jpg';
+
+import './DiscoveryParagraph.scss';
+
+const DiscoveryParagraph: FunctionComponent = () => {
+    const className = 'discovery-paragraph';
+    return (
+        <div className={className}>
+            <Container>
+                <Row>
+                    <Col xs={12} md={12} lg={12}>
+                        <img
+                            src={Elephant}
+                            className={`${className}__image`}
+                        ></img>
+                    </Col>
+                    <Col xs={12} md={12} lg={8}>
+                        <p className={`${className}__paragraph`}>
+                            Lorem ipsum dolor sit amet consectetur adipiscing
+                            elit sodales primis, mollis viverra conubia ligula
```

Comment:
Dar sove ideja, for development purpose generuot tekstui galima https://www.npmjs.com/package/dummy-text-generator sita package pabandyt panaudot, nereiks laikyt niekur ilgu stringu :D

Nu arba savo funkcija kokia nors pasirasyt sitam reikalui :D

Reply:
Man rodos jis nera pritaikytas TS'ui :D, isirasiau ir raso kad nera declarationo generatoriaus. Pameginau su @types negzistuoja modulis toks . Tai arba paciam declarationa suskirt jo. Bet speju tai visai sudetinga ir tingeciau :DD

Reply:
okay, tai laikomes tvarkos rankiniu budu :D